NitroxModel/Helper/WebHelper.cs

```diff
@@ -18,15 +20,15 @@ public static string GetPublicIP()
             }
         }
 
-        public static string GetNitroxLatestVersion()
+        public static Version GetNitroxLatestVersion()
         {
             HttpWebRequest req = WebRequest.Create("https://api.github.com/repos/SubnauticaNitrox/Nitrox/releases/latest") as HttpWebRequest;
             req.Method = "GET";
             req.UserAgent = "Nitrox";
             req.ContentType = "application/json";
             req.Timeout = 1000;
 
-            string version = string.Empty;
+            Version version = new Version();
```

Comment:
It's possible that this instance won't be used so let's init to null and check for null later.
```suggestion
            Version version = null;
```